caikit/runtime/model_management/model_loader.py

```diff
@@ -96,14 +99,26 @@ def load_model(
         return model_builder.build()
 
     def _load_module(
-        self, model_path: str, model_id: str, model_type: str
+        self,
+        model_path: str,
+        model_id: str,
+        model_type: str,
+        finder: Union[None, str, ModelFinderBase],
+        initializer: Union[None, str, ModelInitializerBase],
     ) -> LoadedModel:
         try:
             log.info("<RUN89711114I>", "Loading model '%s'", model_id)
 
+            # Only pass finder/initializer if they have values
+            load_kwargs = {}
+            if finder:
```

Comment:
Should this be an `is None` check to allow empty strings? I think probably not, but I could _maybe_ imagine a case where someone uses an empty key. Probably not worth supporting